CMakeLists.txt

```diff
@@ -183,48 +183,30 @@ if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Intel" AND CMAKE_CXX_COMPILER_VERSION V
 endif()
 
 if (ENABLE_CUDA)
-  set (raja_depends
+  set(raja_depends
     ${raja_depends}
     cuda)
 endif ()
 
 if (ENABLE_CUDA)
   if(ENABLE_EXTERNAL_CUB)
-    find_package(CUB)
-    if (CUB_FOUND)
-      blt_register_library(
-        NAME cub
-        INCLUDES $<BUILD_INTERFACE:${CUB_INCLUDE_DIRS}>)
-      set(raja_depends
-        ${raja_depends}
-        cub)
-    else()
-      message(WARNING "External CUB not found.")
-      set(ENABLE_EXTERNAL_CUB Off)
-    endif()
+    set(raja_depends
+      ${raja_depends}
+      cub)
```

Comment:
Note that I moved this into SetupPackages.cmake so the INCLUDE_DIRS are available in SetupRajaConfig.cmake.